src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
I agree, but then I guess I need to extend the patch to modify all the qlonglong sledgehammer999 used in that class.

I just followed his qlonglong style.